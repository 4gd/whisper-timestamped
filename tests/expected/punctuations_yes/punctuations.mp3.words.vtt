WEBVTT

00:00.380 --> 00:01.300
Dis-moi,

00:01.300 --> 00:01.460
est-ce

00:01.460 --> 00:01.640
que

00:01.640 --> 00:02.000
l'avion

00:02.000 --> 00:02.400
vole?

