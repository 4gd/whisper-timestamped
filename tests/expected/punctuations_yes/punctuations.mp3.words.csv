"Dis-moi,",0.4,1.36
est-ce,1.36,1.52
que,1.52,1.64
l'avion,1.64,2.02
vole?,2.02,2.34
